lib/usd/schemas/wrapTokens.cpp

```diff
@@ -13,52 +13,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// GENERATED FILE.  DO NOT EDIT.
 #include <mayaUsd_Schemas/tokens.h>
 
-#include <boost/python/class.hpp>
+#include <pxr_python.h>
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
-namespace {
-
-// Helper to return a static token as a string.  We wrap tokens as Python
-// strings and for some reason simply wrapping the token using def_readonly
-// bypasses to-Python conversion, leading to the error that there's no
-// Python type for the C++ TfToken type.  So we wrap this functor instead.
-class _WrapStaticToken
-{
-public:
-    _WrapStaticToken(const TfToken* token)
-        : _token(token)
-    {
-    }
-
-    std::string operator()() const { return _token->GetString(); }
-
-private:
-    const TfToken* _token;
-};
-
-template <typename T> void _AddToken(T& cls, const char* name, const TfToken& token)
-{
-    cls.add_static_property(
-        name,
-        boost::python::make_function(
-            _WrapStaticToken(&token),
-            boost::python::return_value_policy<boost::python::return_by_value>(),
-            boost::mpl::vector1<std::string>()));
-}
-
-} // namespace
+#define _ADD_TOKEN(cls, name) \
+    cls.add_static_property(  \
+        #name, +[]() { return MayaUsd_SchemasTokens->name.GetString(); });
```

Comment:
Copied new method (from Pixar) for wrapping static tokens. This removes the need for Boost::mpl